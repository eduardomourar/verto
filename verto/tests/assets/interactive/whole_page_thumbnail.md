{interactive name="binary-cards" type="whole-page" thumbnail="binarycards.png"}
