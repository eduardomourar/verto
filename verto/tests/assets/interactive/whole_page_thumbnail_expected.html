<a class="btn btn-interactive-link" href="{% url 'interactive' name='binary-cards' %}">
<img class="btn-interactive-link-image" src="{% static 'binarycards.png' %}" />
<div class="btn-interactive-whole-page-text">
Click to load binary-cards
</div>
</a>
